METADATA.md

```diff
@@ -7,28 +7,29 @@ The remainder of this document describes _who_ handles metadata _when_ and _how_
 
 ## Files vs. database and time of execution
 
-In all workflows, files are created within ERC in a specific subdirectory `.erc` holding different kinds, formats, or versions of metadata.
+In all workflows files are created within ERC in a specific subdirectory `.erc` holding different kinds, formats, or versions of metadata.
 For ease of access via web API, the information is also stored within the database.
 
 **The files in the compendium are always the normative source of information.**
 
 Mtadata workflows are **conducted on-demand**, i.e. when first needed.
 The brokering output is then stored in respective files and the database.
+By the term _brokering_ the translation from schemaless to schema-specific as well as inter-schema mapping is meant.
 
 ## Metadata extraction and brokering during creation of ERC
 
 **muncher** as the main [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) component for ERC controls the upload workflow.
 The creation from the metadata perspective is as follows:
 
 1. `init` stores the files for a new ERC in a directory.
-1. `extract` uses `metaextract.py` ([docs](https://github.com/o2r-project/o2r-meta#2-metaextract)) to analyse the incoming ERC and creates new files with _raw_ metadata for each of the scanned files.
-This metadata is _schemaless_ and non-semantic.
-  - output file: `.erc/metadata_raw_<file-identifier>.json`
+1. `extract` uses `metaextract.py` ([docs](https://github.com/o2r-project/o2r-meta#2-metaextract)) to analyse the incoming ERC and creates new files with _raw_ metadata for each of the scanned files. Currently the following types of files will be considered: _.r, .rmd, .shp, "bagit.txt"_. Future releases of the extractor will be likely to consider _.tex, .json (geojson), .jp2, .tiff_ and more.
+This raw metadata itself is _schemaless_ and non-semantic. The processed files are in conceptual competetion for the best representative of the working directory's meta information, i.e. there will be only one main output, ideally represented by the most complete set of metadata. By default the competing bits of information will also be preserved in `.erc/metadata_raw_<filename>.json` where _filename_ is an ide based on the original source file.
```

Comment:
it's just a typo, yes. _filename_ has to be an identifier based on the original name. however the original name could be unusable because of:
+ length
+ homonymity from diffrent paths
(+ unsupported chars)

The actual "name" of the file after extraction is yet to be decided. one could go for a timestamp, a checksum or a numeric pattern...